tests/test_align.py

```diff
@@ -93,3 +93,94 @@ def test_astroquery(self):
         rms_y = max(shift_file['col7'])
 
         assert (rms_x <= 0.25 and rms_y <= 0.25)
+
+    def test_align_randomFields(self):
```

Comment:
It might be good to add an option of number of files to select

Reply:
The `test_align_randomFields` is now a wrapper which sets up parameters and a try/except block in order to call `align_randomFields`.  At the moment the parameters (input CSV file, number of final random entries, and the random seed) are hard-coded for testing.